components/textfield/index.css

```diff
@@ -13,23 +13,24 @@ governing permissions and limitations under the License.
 @import '../commons/index.css';
 
 :root {
-  --spectrum-textfield-padding-top: 3px;
-  --spectrum-textfield-padding-bottom: 5px;
+  --spectrum-textfield-quiet-invalid-background-position: 100% 50%;
 
-  /* Todo fix in DNA */
-  --spectrum-textfield-quiet-border-radius: 0;
+  --spectrum-textarea-height: auto;
 
-  --spectrum-textfield-quiet-invalid-background-position: 100% 50%;
+  /* Todo: fix in DNA */
+  --spectrum-textarea-min-height: var(--spectrum-global-dimension-size-700);
```

Comment:
In design workshop, we decided it should be a single line by default instead of 2 lines.